### ISSUE TITLE
about train_beam_size in SCST training stage

### ISSUE THREAD

**Issue by wanboyang** [ruotianluo/ImageCaptioning.pytorch#121]
I use the train_beam_size shell script recommended from [https://github.com/ruotianluo/ImageCaptioning.pytorch/blob/master/ADVANCED.md](url) to train updown model.
here are my shell scripts:
```
python tools/train.py --id updown --cfg configs/updown/updown.yml

bash scripts/copy_model.sh updown updown_rl_beam1

python tools/train.py --id updown_rl_beam1 --cfg configs/updown/updown_sc.yml --train_sample_method greedy --train_beam_size 5 --scheduled_sampling_start -1
```
However,  RL train performs bad performances:

**Comment by wanboyang** [comment]
Here are Curves of CIDEr Scores, Losses and Rewards. Yellow line and blue line indicates no train_beam_size and train beam size, respectively.
![QQ截图20210217195533](https://user-images.githubusercontent.com/23290289/108201162-67f46f80-715a-11eb-8a0d-9828e8de4470.jpg)
![loss](https://user-images.githubusercontent.com/23290289/108201172-6c208d00-715a-11eb-9c9f-d72cbdc48169.jpg)
![reward](https://user-images.githubusercontent.com/23290289/108201180-6f1b7d80-715a-11eb-920e-1dfd95ca020f.jpg)

**Comment by ruotianluo** [comment]
This is expected. You need to use a lower learning rate.

Even though, the performance is better with random sampling. (I didn't suggest to use it, I just provided this option)

**Comment by wanboyang** [comment]
Thanks for replying.
I use scripts :
`python train.py --id fc_tdsc --cfg configs/fc_rl.yml --train_sample_method greedy --train_beam_size 5 --max_epochs 30 --learning_rate 5e-6`
the RL training stage starts at 350k step, the performance corrupts.
![image](https://user-images.githubusercontent.com/23290289/108450099-8dd75c80-729f-11eb-8135-5db15f5199c6.png)
what is learning rate you recommende?

**Comment by wanboyang** [comment]
I will try lower learning rates
